sympy/polys/domains/domain.py

```
@@ -1012,6 +1013,11 @@ def poly_ring(self, *symbols: str | Expr, order: str | MonomialOrder = lex) -> P
         from sympy.polys.domains.polynomialring import PolynomialRing
         return PolynomialRing(self, symbols, order)
 
+    def power_series_ring(self, *symbols: str | Expr, prec: int = 6) -> PowerSeriesRing:
+        """Returns a Power Series ring over ring or field. """
+        from sympy.polys.domains.powerseriesring import PowerSeriesRing
+        return PowerSeriesRing(self, symbols[0], prec)
```

This should raise an error if multiple symbols are passed.